GWAS/Hudson_plot.ipynb

```diff
@@ -278,7 +278,11 @@
     "  \n",
     "    # Create the dataframes with useful info\n",
     "    qqdat1 <- data.frame(SNP=gwas.1$${snp},CHR=gwas.1$CHR,POS=gwas.1$${bp}, pvalue=gwas.1$${pval},length.out=nrow(gwas.1), each=1)\n",
+    "    qqdat1 <- qqdat1 %>%\n",
+    "        filter(${pval} < 0.05)\n",
```

Comment:
@dianacornejo what happened to `pval_filter`? should be `${pval} < ${pval_filter}` right?

Reply:
@gaow the pval filter was to select the snps to be highlighted, whereas this other filter is to thin the plot

Reply:
I see @dianacornejo its up to you if you want to make this 0.05 another parameter. If so please make the change. Otherwise just tell me No and i'll merge

Reply:
@gaow I don't think it is necessary to create another parameter I think 0.05 is good enough. :)